models/general/knowledge/en/weather_information.txt

```diff
@@ -1,60 +1,161 @@
 #Weather Api 
 
+hashtags * *
+!console:The following hashtags are common with $1$: $k1$ ($v1$ times), $k2$ ($v2$ times), $k3$ ($v3$ times)
+{
+"url":"http://api.loklak.org/api/search.json?q=$2$&source=cache&count=0&fields=hashtags&limit=6",
+"path":"$.aggregations.hashtags"
+}
+eol
+
 
-# kelvin to celsius conversion
-# we need that for the weather api
-kelvin to celsius *
+mentions * *
+!console:The following users are mentioned with $1$: @$k0$ ($v0$ times), @$k1$ ($v1$ times), @$k2$ ($v2$ times)
+{
+"url":"http://api.loklak.org/api/search.json?q=$2$&source=cache&count=0&fields=mentions&limit=6",
+"path":"$.aggregations.mentions"
+}
+eol
+
```

Comment:
Above skills do not give weather information, maybe we should change it to some other file?